src/celesto/sdk/client.py

```
@@ -5,24 +5,30 @@
 from .types import ComputerTerminalSessionInfo
 
-_BASE_URL = os.environ.get("CELESTO_BASE_URL", "https://api.celesto.ai/v1")
+__all__ = [
+    "_BASE_URL",
+    "_BaseClient",
+    "_BaseConnection",
+    "_CelestoClient",
+    "Computers",
+    "Deployment",
+    "GateKeeper",
+]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)